doc/2/guides/upgrade-kuzzle/changes/index.md

```diff
@@ -160,6 +182,8 @@ It accepts the same arguments as the `kuzzle start` command from the CLI.
 
   - `server.protocols.socketio.enable` is now `false`, deactivating the Socket.io protocol by default
   - `services.storage.commonMapping.dynamic` is now `false` by default, meaning that Elasticsearch will not infer mapping of new introduced fields
+  - `security.standard.roles.default.controllers.server.actions` is now `{ publicApi: true }` instead of `{ info: true }`
+  - `security.standard.roles.anonymous.controllers.server.actions` is now `{ publicApi: true }` instead of `{ info: true }`
```

Comment:
This should be added to the upgrade script: https://github.com/kuzzleio/kuzzle/blob/2-dev/bin/.upgrades/versions/v1/checkConfiguration.js#L28

Reply:
Why ? I didn't rename the entry, I change the value of default rights

Reply:
isn't `security.standard.roles.anonymous.controllers.server.actions.info` obsolete now?

Reply:
Ah, my bad.